spec/integration/post_index_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index', type: :system do
+  user = User.first
+  post = Post.find(1)
+  comment = Comment.create(user:, post:, text: 'some text')
+  # it 'can see the users profile picture' do
+  #     visit user_post_path(user, post)
+  #     expect(page.body).to include('img_placeholder')
+  # end
```

Comment:
- [ ] Please kindly remove `commented code` from your `code base`, as it makes your `code base` dirty and less readable. To make sure that your codebase is always clean, always remember to remove commented-codes.